apm-sniffer/config/agent.config

```diff
@@ -93,3 +93,7 @@ logging.level=${SW_LOGGING_LEVEL:INFO}
 
 # Exclude activated plugins
 # plugin.exclude_plugins=${SW_EXCLUDE_PLUGINS:""}
+
+# Optional Plugins config
```

Comment:
```suggestion
```

Many plugins above are optional too. If you want to indicate some are optional, which requires activating first, then we could work on this in a separate PR.